.rubocop.yml

```diff
@@ -0,0 +1,20 @@
+inherit_from: .rubocop_todo.yml
+
+HashSyntax:
+  EnforcedStyle: hash_rockets
```

Comment:
All these rules should be changed to the update names - e.g. https://rubocop.readthedocs.io/en/latest/cops_style/#stylehashsyntax